doc/dev/design/FABRID.rst

```diff
@@ -0,0 +1,605 @@
+********
+FABRID
+********
+.. _fabrid-design:
+
+- Author: Justin Rohrer, Jelte van Bommel, Marc Odermatt, Marc Wyss, Cyrill Krähenbühl, Juan A. García-Pardo
```

Comment:
Please make "Author", "Last updated", and "Discussion at" **bold**.
Also  "Last Updated" -> "Last updated"